packages/teleport-plugin-jsx-head-config/src/index.ts

```diff
@@ -72,7 +72,7 @@ export const createJSXHeadConfigPlugin: ComponentPluginFactory<JSXHeadPluginConf
       rootElement.children.unshift(headConfigTag)
 
       dependencies[configTagIdentifier] = {
-        type: 'library',
+        type: 'package',
         path: configTagDependencyPath,
```

Comment:
`react-helmet` is a external package and need to be added to `package.json`. This should happen only if the package is used. Currently by default we added in templates which is not needed. And this is a requirement based usage and not a core part of react. So, we need to switch to `package` according to conventions.